js/src/csp/CSPVisualizer.ts

```diff
@@ -3,7 +3,7 @@ import { timeout } from "d3";
 import * as Analytics from "../Analytics";
```

Comment:
The widgets don't seem to work for me on your branch after I pulled the latest commit. 

Before the last commit I had tried to run a csp consistency example and the rectangular nodes didn't seem to work.

Reply:
Fixed.